flopy/mf6/mfsimbase.py

```diff
@@ -689,9 +692,60 @@ def exchange_files(self):
         """
         return self._exchange_files.values()
 
+    @property
+    def package_key_dict(self):
+        warnings.warn(
+            "package_key_dict has been deprecated, use "
+            "package_type_dict instead",
+            category=DeprecationWarning,
+        )
+        return self._package_container.package_type_dict
+
+    @property
+    def package_dict(self):
+        """Returns a copy of the package name dictionary."""
+        warnings.warn(
+            "This method is for internal use only and will be deprecated.",
+            category=DeprecationWarning,
+        )
+        return self._package_container.package_dict
+
+    @property
+    def package_names(self):
+        """Returns a list of package names."""
+        warnings.warn(
+            "This method is for internal use only and will be deprecated.",
+            category=DeprecationWarning,
+        )
+        return self._package_container.package_names
+
+    @property
+    def package_type_dict(self):
+        warnings.warn(
+            "This method is for internal use only and will be deprecated.",
+            category=DeprecationWarning,
+        )
+        return self._package_container.package_type_dict
+
+    @property
+    def package_name_dict(self):
+        warnings.warn(
+            "This method is for internal use only and will be deprecated.",
+            category=DeprecationWarning,
+        )
+        return self._package_container.package_name_dict
+
+    @property
+    def package_filename_dict(self):
+        warnings.warn(
+            "This method is for internal use only and will be deprecated.",
+            category=DeprecationWarning,
+        )
+        return self._package_container.package_filename_dict
+
     @staticmethod
     def load(
-        cls_child,
+        cls_child: Type["MFSimulationBase"],
```

Comment:
I don't think `typing.Type` is needed since the builtin `type` supports subscripts https://docs.python.org/3/library/typing.html#typing.Type